### ISSUE TITLE
Improve titles for Cypress accessibility tests.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#10323]
## Description
See e.g. [this comment](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/10279#issuecomment-1215991747) on a recent PR:

```
    1) Login page has no detectable accessibility violations on load.


  0 passing (2s)
  1 failing

  1) Component accessibility test
       Login page has no detectable accessibility violations on load.:
     AssertionError: 1 accessibility violation was detected: expected 1 to equal 0
      at Context.eval (https://pr10279-jphvf6vnpqbuaeyfr8pdufwlni8aumug.ci.cms.va.gov/__cypress/tests?p=tests/cypress/support/index.js:9682:20)
```

This kinda makes it sound like it is stating that there are no detectable accessibility violations on load, and then it says there's one.  This has confused multiple developers; I should look and see if there is a better pattern for naming these, or for renaming them if and when they fail.

## Acceptance Criteria
- [ ] Failed tests make it clear that an expectation or assertion was violated, or express the actual error condition, rather than appearing to state something that isn't true.
